### PR TITLE
New: cursor and autoFocus

### PR DESCRIPTION
**Pull request by Rivki7**
Hi, 
I added cursor pointer fot the buttons and auto focus in the forms in the first screen of the app.

### CODE REVIEW

**Comment by baruchiro** [comment]
Use the "Hide whitespace" to easily review the real change:
image

The rest of the change is because we added a `lint-staged` script which will run on your changed files and format them before the commit. Now when someone touches on a file that wasn't formatted before, he will take the new format with him.

File: ui-react/src/components/topBar/ReportProblemModal.tsx [hunk 1]
```diff
@@ -196,6 +195,7 @@ function ReportProblemModal({ show, onClose }: ReportProblemModalProps) {
                   value={form.title}
                   isInvalid={!!errors.title}
                   onChange={(e) => setField('title', e.target.value)}
+                  autoFocus
```
Comment (baruchiro) [inline ui-react/src/components/topBar/ReportProblemModal.tsx:198]: This is the change

File: ui-react/src/components/accounts/EditImporter.tsx [hunk 2]
```diff
@@ -54,27 +65,54 @@ export default function EditImporter({
...
+                  controlId={loginField}
+                >
+                  <Form.Control
+                    placeholder={LOGIN_FIELD_DISPLAY_NAMES[loginField]}
+                    type={loginField === 'password' ? 'password' : ''}
+                    value={loginFields[loginField]}
+                    onChange={(event) =>
+                      onLoginFieldChanged(loginField, event.target.value)
+                    }
+                    autoFocus={index === 0}
```
Comment (baruchiro) [inline ui-react/src/components/accounts/EditImporter.tsx:91]: This is the change

File: ui-react/src/components/accounts/Account.tsx [hunk 3]
```diff
@@ -8,35 +8,49 @@ import StatusIndicator from './StatusIndicator';
...
-}: AccountProps) {
+export default function Account({ account, actionButtons }: AccountProps) {
   const containerStyles = [styles.container];
 
-  const badgeNumberLog = account.logs.find((log) => log.originalEvent && log.originalEvent.exportedTransactionsNum > 0);
+  const badgeNumberLog = account.logs.find(
+    (log) => log.originalEvent && log.originalEvent.exportedTransactionsNum > 0,
+  );
   if (!account.active) containerStyles.push(styles.notActive);
+  if (!actionButtons) containerStyles.push(styles.pointer);
```
Comment (baruchiro) [inline ui-react/src/components/accounts/Account.tsx:26]: This is the change.

Although I'm not sute why you need this if, can you explain?

**Comment by baruchiro** [comment]
Do you think you can cursor these too?
image

**Comment by Rivki7** [comment]
> Use the "Hide whitespace" to easily review the real change: image
> 
> The rest of the change is because we added a `lint-staged` script which will run on your changed files and format them before the commit. Now when someone touches on a file that wasn't formatted before, he will take the new format with him.

Good to know that, thanks!

**Comment by Rivki7** [comment]
> Do you think you can cursor these too? image

I added that to the settings icon now (and also added autoFocus to the setting modal first text area). 
I tried to add that to the switch input as well but it is too complicated because of bootstrap. I consulted with Gemini and gpt and couldn't find a simple way to do that without changing the form structure.

File: ui-react/src/components/Body.tsx [hunk 4]
```diff
@@ -65,58 +69,112 @@ const Body = ({ scrape }: BodyProps) => {
...
+          disabled={store.isScraping}
+        >
+          הפעל
+        </Button>
+        <Image
+          src={settingsIcon}
+          onClick={() => showModal(null, ModalStatus.GENERAL_SETTINGS)}
+          className={styles.pointer}
+        />
         <Form.Check
```
Comment (baruchiro) [inline ui-react/src/components/Body.tsx:155]: This is the change

**Review by baruchiro — APPROVED** [review]
Thank you!

ThankYouThanksGIF

**Pull request merged**